src/services/contentful.ts

```diff
@@ -1,8 +1,26 @@
 import { createClient, CreateClientParams } from "contentful";
+import { contenfulImageFormated } from "src/utils/formating";
 
 const params = {
   space: process.env.CONTENTFUL_SPACE_ID,
   accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
 } as CreateClientParams;
 
-export const client = createClient(params);
+export const getPosts = async () => {
+  const client = createClient(params);
+  const { items } = await client.getEntries({
+    content_type: "post",
+  });
+  const posts: Array<any> = items.map((post: any): Object => {
```

Comment:
Maybe type this array as well?

Please note this would be a different interface than the one above. This interface should represent data types coming from Contentful.